klippy/clocksync.py

```diff
@@ -61,9 +61,11 @@ def _get_clock_event(self, eventtime):
         self.queries_pending += 1
         # Use an unusual time for the next event so clock messages
         # don't resonate with other periodic events.
-        return eventtime + .9839
+        next_eventtime = eventtime + .9839
+        self.reactor.update_timer(self.get_clock_timer, next_eventtime)
+        return next_eventtime
```

Comment:
I don't believe that this change makes a functional difference.  If we look at `reactor.update_timer()` we can see that it updates the timer handler's `waketime` and the reactor's `_next_timer`:

https://github.com/Klipper3d/klipper/blob/16b4b6b302ac3ffcd55006cd76265aad4e26ecc8/klippy/reactor.py#L120-L122

Looking at what `reactor.check_timers()` does with the callback's return value, we can see it also updates the timer hander's `waketime` and the reactor's `_next_timer`:

https://github.com/Klipper3d/klipper/blob/16b4b6b302ac3ffcd55006cd76265aad4e26ecc8/klippy/reactor.py#L154-L163